### PR TITLE
Add pam option file

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -10,12 +10,18 @@
...
-- name: set nginx config
+- name: Set nginx config
   template:
     src: templates/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
   register: nginx_reload
 
+- name: Place template pam nginx file
+  template:
+    src: "templates/pam_service_file.j2"
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: why is this quoted?

File: tasks/main.yml [hunk 2]
```diff
@@ -10,12 +10,18 @@
...
+    dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: documentation? Is it mandatory? It should be, or its definition should be checked.

File: templates/pam_service_file.j2 [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{% for rule in nginx_pam_rules %}
```
Comment (SlothOfAnarchy) [inline templates/pam_service_file.j2:2]: ansible_managed

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (Scriptkiddi) [reply tasks/main.yml]: fixed

File: tasks/main.yml (hunk 2 above)
Comment (Scriptkiddi) [reply tasks/main.yml]: see the when statment below

File: templates/pam_service_file.j2 (hunk 3 above)
Comment (Scriptkiddi) [reply templates/pam_service_file.j2:2]: fixed

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rereview

**Review by dasJ — DISMISSED** [review]

File: tasks/main.yml [hunk 4]
```diff
@@ -10,12 +10,18 @@
...
+- name: Set nginx config
   template:
     src: templates/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
   register: nginx_reload
 
+- name: Place template pam nginx file
+  template:
+    src: templates/pam_service_file.j2
+    dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: The var is not actually mandatory, as described in readme

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 4 above)
Comment (Scriptkiddi) [reply tasks/main.yml]: what are you talking about

Comment (SlothOfAnarchy) [reply tasks/main.yml]: replace
```
dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```
by
```
dest: "/etc/pam.d/{{ nginx_pam_service_name | mandatory }}"
```

Comment (Scriptkiddi) [reply tasks/main.yml]: done

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**